waku/waku_test.go

```diff
@@ -1491,15 +1533,21 @@ func TestWakuTimeDesyncEnvelopeIgnored(t *testing.T) {
 	}
 	rw1, rw2 := p2p.MsgPipe()
 	defer func() {
-		rw1.Close()
-		rw2.Close()
+		if err := rw1.Close(); err != nil {
+			t.Errorf("error closing MsgPipe, '%s'", err)
+		}
+		if err := rw2.Close(); err != nil {
+			t.Errorf("error closing MsgPipe, '%s'", err)
+		}
```

Comment:
[Error handling is good](https://github.com/status-im/status-go/pull/1957/files#r417328424)